### PR TITLE
Improve CF documentation

### PR DESCRIPTION
**Pull request by lukemorton**
These changes help bring the CF doc in line with current practice.

### CODE REVIEW

File: policies/continuous_feedback.md [hunk 1]
```diff
@@ -1,108 +1,126 @@
...
-Our continuous feedback sessions involve 2 people, yourself and a facilitator. The facilitator will either be a colleague or a director. Their role is to help you make sense of your feedback and to help you track your progress against goals you've set yourself.
+## How does it work?
+
+Our continuous feedback sessions involve 2 people, yourself and a facilitator. The aim of a continuous feedback session is to:
+
+ - Receive and acknowledge feedback
+ - Setting and reviewing longer term goals
+ - Setting and reviewing short term goals
+
+The facilitator will either be a colleague or a director. Their role is to help you make sense of your feedback and to help you track your progress against goals you've set yourself.
```
Comment (craigjbass) [inline policies/continuous_feedback.md]: Don't like the fact that the wording "colleague or a director" implies at first glance that directors are not colleagues

File: policies/continuous_feedback.md [hunk 2]
```diff
@@ -1,108 +1,126 @@
...
+ - Given with positive intent to help to be more effective in future
+ - Close to the event
+ - Don’t give feedback when angry or upset
+ - Leverage the SAID model
+ - Two questions to give effective feedback:
+    - What does this person do well that makes them effective? Be specific and share with them why it’s impactful
+    - What is one thing, looking forward, they could change or do more of that would make them more effective?
 
-###Receiving feedback
+If you have feedback concerning more than one person, submit it to both people. Depending on the issue/feedback, calling a having a separate meeting/discussion outside of CF might be a better solution.
```
Comment (craigjbass) [inline policies/continuous_feedback.md]: Could refer to retros here

Comment (craigjbass) [inline policies/continuous_feedback.md]: Although that may not help intra-team feedback :+1:

File: policies/continuous_feedback.md (hunk 1 above)
Comment (lukemorton) [inline policies/continuous_feedback.md]: Have improved it I think

**Pull request merged**